gnomad_constraint/plots/generate_constraint_plots.R

```diff
@@ -243,3 +309,226 @@ for (version in names(datasets)) {
     ggsave(p, filename = plot_path, dpi = 300, width = 8, height = 6, units = "in")
   }
 }
+
+
+####################################################################
+####################################################################
+# Plot LOEUF decile change from v2 to v4
+####################################################################
+####################################################################
+# Filter to MANE Select Ensembl transcripts that are present in both v2 and v4
```

Comment:
Since this is filtering transcripts, should the `filter_transcripts` be modified to support this type of filtering?

Also, this is only filtering on v4 constraint flags and mane select, just making sure that is what is wanted

Reply:
yeah, didn't want any v2 specific filters here

filter_transcripts is version specific but i can simplify this part more